napari/_qt/containers/_layer_delegate.py

```diff
@@ -56,6 +60,27 @@
     from ...components.layerlist import LayerList
 
 
+@inject_napari_dependencies
+def show_layer_info(ll: LayerList):
+    layer = ll.selection.active
+    dlg = QtLayerInfoDialog(layer=layer)
+    pos = QCursor.pos()
+    dlg.move(pos)
+    dlg.exec_()
+
+
+_QT_LAYER_ACTIONS = [
+    {
+        'napari:show_layer_info': {
+            'description': trans._('Show layer info'),
+            'action': show_layer_info,
+            'enable_when': LLCK.num_selected_layers == 1,
+            'show_when': True,
+        }
+    }
+]
```

Comment:
```suggestion
_LAYER_ACTIONS.append( 
    {
        'napari:show_layer_info': {
            'description': trans._('Show layer info'),
            'action': show_layer_info,
            'enable_when': LLCK.num_selected_layers == 1,
            'show_when': True,
        }
    }
)
```

plus import `_LAYER_ACTIONS` at begin of file.